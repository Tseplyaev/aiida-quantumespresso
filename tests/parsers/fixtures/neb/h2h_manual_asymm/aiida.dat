      0.0000000000      0.0000000000      2.1764284244
      0.3072564850     -0.1343028807      0.1582894289
      0.6935429724     -0.1339354387      0.0909790512
      1.0000000000     -0.0000000001      2.1764222216
