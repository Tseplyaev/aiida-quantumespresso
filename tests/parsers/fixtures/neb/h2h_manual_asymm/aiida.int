      0.0000000000      0.0000000000
      0.0040000000     -0.0000447996
      0.0080000000     -0.0001780358
      0.0120000000     -0.0003979644
      0.0160000000     -0.0007028413
      0.0200000000     -0.0010909225
      0.0240000000     -0.0015604637
      0.0280000000     -0.0021097209
      0.0320000000     -0.0027369500
      0.0360000000     -0.0034404069
      0.0400000000     -0.0042183475
      0.0440000000     -0.0050690276
      0.0480000000     -0.0059907031
      0.0520000000     -0.0069816300
      0.0560000000     -0.0080400641
      0.0600000000     -0.0091642614
      0.0640000000     -0.0103524776
      0.0680000000     -0.0116029687
      0.0720000000     -0.0129139907
      0.0760000000     -0.0142837993
      0.0800000000     -0.0157106505
      0.0840000000     -0.0171928001
      0.0880000000     -0.0187285041
      0.0920000000     -0.0203160184
      0.0960000000     -0.0219535988
      0.1000000000     -0.0236395012
      0.1040000000     -0.0253719815
      0.1080000000     -0.0271492957
      0.1120000000     -0.0289696995
      0.1160000000     -0.0308314489
      0.1200000000     -0.0327327998
      0.1240000000     -0.0346720081
      0.1280000000     -0.0366473297
      0.1320000000     -0.0386570204
      0.1360000000     -0.0406993361
      0.1400000000     -0.0427725328
      0.1440000000     -0.0448748663
      0.1480000000     -0.0470045925
      0.1520000000     -0.0491599674
      0.1560000000     -0.0513392467
      0.1600000000     -0.0535406864
      0.1640000000     -0.0557625424
      0.1680000000     -0.0580030706
      0.1720000000     -0.0602605268
      0.1760000000     -0.0625331670
      0.1800000000     -0.0648192471
      0.1840000000     -0.0671170228
      0.1880000000     -0.0694247502
      0.1920000000     -0.0717406851
      0.1960000000     -0.0740630835
      0.2000000000     -0.0763902011
      0.2040000000     -0.0787202939
      0.2080000000     -0.0810516178
      0.2120000000     -0.0833824286
      0.2160000000     -0.0857109823
      0.2200000000     -0.0880355348
      0.2240000000     -0.0903543419
      0.2280000000     -0.0926656596
      0.2320000000     -0.0949677436
      0.2360000000     -0.0972588500
      0.2400000000     -0.0995372346
      0.2440000000     -0.1018011533
      0.2480000000     -0.1040488620
      0.2520000000     -0.1062786165
      0.2560000000     -0.1084886729
      0.2600000000     -0.1106772869
      0.2640000000     -0.1128427144
      0.2680000000     -0.1149832114
      0.2720000000     -0.1170970337
      0.2760000000     -0.1191824372
      0.2800000000     -0.1212376779
      0.2840000000     -0.1232610116
      0.2880000000     -0.1252506941
      0.2920000000     -0.1272049815
      0.2960000000     -0.1291221295
      0.3000000000     -0.1310003941
      0.3040000000     -0.1328380311
      0.3080000000     -0.1346333635
      0.3120000000     -0.1363876014
      0.3160000000     -0.1381019206
      0.3200000000     -0.1397764159
      0.3240000000     -0.1414111822
      0.3280000000     -0.1430063144
      0.3320000000     -0.1445619073
      0.3360000000     -0.1460780558
      0.3400000000     -0.1475548549
      0.3440000000     -0.1489923993
      0.3480000000     -0.1503907840
      0.3520000000     -0.1517501038
      0.3560000000     -0.1530704536
      0.3600000000     -0.1543519282
      0.3640000000     -0.1555946226
      0.3680000000     -0.1567986317
      0.3720000000     -0.1579640502
      0.3760000000     -0.1590909731
      0.3800000000     -0.1601794953
      0.3840000000     -0.1612297116
      0.3880000000     -0.1622417168
      0.3920000000     -0.1632156060
      0.3960000000     -0.1641514739
      0.4000000000     -0.1650494155
      0.4040000000     -0.1659095255
      0.4080000000     -0.1667318989
      0.4120000000     -0.1675166306
      0.4160000000     -0.1682638154
      0.4200000000     -0.1689735482
      0.4240000000     -0.1696459238
      0.4280000000     -0.1702810372
      0.4320000000     -0.1708789833
      0.4360000000     -0.1714398568
      0.4400000000     -0.1719637528
      0.4440000000     -0.1724507660
      0.4480000000     -0.1729009913
      0.4520000000     -0.1733145236
      0.4560000000     -0.1736914578
      0.4600000000     -0.1740318887
      0.4640000000     -0.1743359113
      0.4680000000     -0.1746036204
      0.4720000000     -0.1748351108
      0.4760000000     -0.1750304775
      0.4800000000     -0.1751898154
      0.4840000000     -0.1753132192
      0.4880000000     -0.1754007839
      0.4920000000     -0.1754526044
      0.4960000000     -0.1754687755
      0.5000000000     -0.1754493921
      0.5040000000     -0.1753945491
      0.5080000000     -0.1753043414
      0.5120000000     -0.1751788638
      0.5160000000     -0.1750182112
      0.5200000000     -0.1748224784
      0.5240000000     -0.1745917605
      0.5280000000     -0.1743261521
      0.5320000000     -0.1740257483
      0.5360000000     -0.1736906438
      0.5400000000     -0.1733209336
      0.5440000000     -0.1729167126
      0.5480000000     -0.1724780755
      0.5520000000     -0.1720051174
      0.5560000000     -0.1714979329
      0.5600000000     -0.1709566172
      0.5640000000     -0.1703812649
      0.5680000000     -0.1697719710
      0.5720000000     -0.1691288304
      0.5760000000     -0.1684519379
      0.5800000000     -0.1677413884
      0.5840000000     -0.1669972768
      0.5880000000     -0.1662196979
      0.5920000000     -0.1654087467
      0.5960000000     -0.1645645180
      0.6000000000     -0.1636871067
      0.6040000000     -0.1627766077
      0.6080000000     -0.1618331157
      0.6120000000     -0.1608567258
      0.6160000000     -0.1598475328
      0.6200000000     -0.1588056315
      0.6240000000     -0.1577311169
      0.6280000000     -0.1566240838
      0.6320000000     -0.1554846271
      0.6360000000     -0.1543128416
      0.6400000000     -0.1531088223
      0.6440000000     -0.1518726640
      0.6480000000     -0.1506044615
      0.6520000000     -0.1493043099
      0.6560000000     -0.1479723039
      0.6600000000     -0.1466085383
      0.6640000000     -0.1452131082
      0.6680000000     -0.1437861083
      0.6720000000     -0.1423276336
      0.6760000000     -0.1408377789
      0.6800000000     -0.1393166391
      0.6840000000     -0.1377643090
      0.6880000000     -0.1361808836
      0.6920000000     -0.1345664577
      0.6960000000     -0.1329173622
      0.7000000000     -0.1312197708
      0.7040000000     -0.1294740501
      0.7080000000     -0.1276820959
      0.7120000000     -0.1258458038
      0.7160000000     -0.1239670696
      0.7200000000     -0.1220477890
      0.7240000000     -0.1200898576
      0.7280000000     -0.1180951712
      0.7320000000     -0.1160656255
      0.7360000000     -0.1140031161
      0.7400000000     -0.1119095389
      0.7440000000     -0.1097867894
      0.7480000000     -0.1076367634
      0.7520000000     -0.1054613566
      0.7560000000     -0.1032624647
      0.7600000000     -0.1010419834
      0.7640000000     -0.0988018084
      0.7680000000     -0.0965438354
      0.7720000000     -0.0942699601
      0.7760000000     -0.0919820782
      0.7800000000     -0.0896820855
      0.7840000000     -0.0873718775
      0.7880000000     -0.0850533501
      0.7920000000     -0.0827283989
      0.7960000000     -0.0803989196
      0.8000000000     -0.0780668080
      0.8040000000     -0.0757339597
      0.8080000000     -0.0734022704
      0.8120000000     -0.0710736359
      0.8160000000     -0.0687499518
      0.8200000000     -0.0664331139
      0.8240000000     -0.0641250178
      0.8280000000     -0.0618275593
      0.8320000000     -0.0595426340
      0.8360000000     -0.0572721377
      0.8400000000     -0.0550179661
      0.8440000000     -0.0527820148
      0.8480000000     -0.0505661796
      0.8520000000     -0.0483723562
      0.8560000000     -0.0462024402
      0.8600000000     -0.0440583275
      0.8640000000     -0.0419419136
      0.8680000000     -0.0398550943
      0.8720000000     -0.0377997653
      0.8760000000     -0.0357778223
      0.8800000000     -0.0337911610
      0.8840000000     -0.0318416771
      0.8880000000     -0.0299312663
      0.8920000000     -0.0280618243
      0.8960000000     -0.0262352468
      0.9000000000     -0.0244534295
      0.9040000000     -0.0227182682
      0.9080000000     -0.0210316585
      0.9120000000     -0.0193954961
      0.9160000000     -0.0178116767
      0.9200000000     -0.0162820960
      0.9240000000     -0.0148086498
      0.9280000000     -0.0133932338
      0.9320000000     -0.0120377436
      0.9360000000     -0.0107440749
      0.9400000000     -0.0095141234
      0.9440000000     -0.0083497850
      0.9480000000     -0.0072529551
      0.9520000000     -0.0062255297
      0.9560000000     -0.0052694043
      0.9600000000     -0.0043864746
      0.9640000000     -0.0035786365
      0.9680000000     -0.0028477855
      0.9720000000     -0.0021958173
      0.9760000000     -0.0016246278
      0.9800000000     -0.0011361125
      0.9840000000     -0.0007321673
      0.9880000000     -0.0004146877
      0.9920000000     -0.0001855695
      0.9960000000     -0.0000467084
      1.0000000000     -0.0000000001
