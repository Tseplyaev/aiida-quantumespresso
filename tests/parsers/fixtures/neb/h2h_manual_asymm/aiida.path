RESTART INFORMATION
      20
      20
       0
NUMBER OF IMAGES
   4
ENERGIES, POSITIONS AND GRADIENTS
Image:    1
       -1.6399117415
     -4.566700090012      0.000000000000      0.000000000000     -0.000384669442     -0.000000000000     -0.000000000000  1  0  0
      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000     -0.000000000000  0  0  0
      1.557766760017      0.000000000000      0.000000000000      0.042324797747     -0.000000000000     -0.000000000000  1  0  0
Image:    2
       -1.6448472818
     -2.918728029018      0.000000000000      0.000000000000     -0.003003122240     -0.000000000000     -0.000000000000
      0.000000000000      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000
      1.455649817851      0.000000000000      0.000000000000      0.000813039804     -0.000000000000     -0.000000000000
Image:    3
       -1.6448337786
     -1.450763358183      0.000000000000      0.000000000000      0.000507664554     -0.000000000000     -0.000000000000
      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000     -0.000000000000
      2.923343286087      0.000000000000      0.000000000000      0.002845926010     -0.000000000000     -0.000000000000
Image:    4
       -1.6399117415
     -1.557766760017      0.000000000000      0.000000000000     -0.042324677122     -0.000000000000     -0.000000000000
      0.000000000000      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000
      4.566700090012      0.000000000000      0.000000000000      0.000384671895     -0.000000000000     -0.000000000000
