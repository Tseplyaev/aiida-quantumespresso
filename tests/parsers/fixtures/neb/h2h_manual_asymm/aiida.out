
     Program NEB v.6.4.1 starts on  9Sep2019 at 21: 1: 6 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes

     No input file found, assuming nothing to parse
     Searching argument -input_images or --input_images
     Reading input from pw_1.in
     Reading input from pw_2.in

     initial path length           =  4.2553 bohr
     initial inter-image distance  =  1.4184 bohr

     string_method                 =    neb
     restart_mode                  =    from_scratch
     opt_scheme                    =    broyden
     num_of_images                 =    4
     nstep_path                    =    20
     CI_scheme                     =    manual
     first_last_opt                =    F
     use_freezing                  =    F
     ds                            =    2.0000 a.u.
     k_max                         =    0.3000 a.u.
     k_min                         =    0.2000 a.u.
     suggested k_max               =    0.1542 a.u.
     suggested k_min               =    0.1028 a.u.
     path_thr                      =    0.0500 eV / A

     list of climbing images :   2,

     ------------------------------ iteration   1 ------------------------------

     tcpu =      0.2    self-consistency for image   1
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      0.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      1.6    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      2.4    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   2.014397 eV
     activation energy (<-) =   2.014397 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -42.6098713            4.076889            F
         3        -42.6098713            3.740892            F
         4        -44.6242679            2.176422            T

     path length          =  4.255 bohr
     inter-image distance =  1.418 bohr

     ------------------------------ iteration   2 ------------------------------

     tcpu =      3.1    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      4.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   1.715272 eV
     activation energy (<-) =   1.715272 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -42.9089963            3.851001            F
         3        -42.9477006            3.713106            F
         4        -44.6242679            2.176422            T

     path length          =  4.301 bohr
     inter-image distance =  1.434 bohr

     ------------------------------ iteration   3 ------------------------------

     tcpu =      5.0    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      5.8    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   1.171369 eV
     activation energy (<-) =   1.171369 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -43.4528989            2.886442            F
         3        -43.8525050            3.259646            F
         4        -44.6242679            2.176422            T

     path length          =  4.513 bohr
     inter-image distance =  1.504 bohr

     ------------------------------ iteration   4 ------------------------------

     tcpu =      6.7    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      7.5    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.486496 eV
     activation energy (<-) =   0.486496 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.1377722            3.064123            F
         3        -44.5828952            2.105733            F
         4        -44.6242679            2.176422            T

     path length          =  4.909 bohr
     inter-image distance =  1.636 bohr

     ------------------------------ iteration   5 ------------------------------

     tcpu =      8.3    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      9.2    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7362732            1.903314            F
         3        -44.6393313            4.754447            F
         4        -44.6242679            2.176422            T

     path length          =  5.473 bohr
     inter-image distance =  1.824 bohr

     ------------------------------ iteration   6 ------------------------------

     tcpu =      9.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     10.6    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.236059 eV
     activation energy (<-) =   0.236059 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.3882089            2.697623            F
         3        -44.6207233            1.562016            F
         4        -44.6242679            2.176422            T

     path length          =  5.016 bohr
     inter-image distance =  1.672 bohr

     ------------------------------ iteration   7 ------------------------------

     tcpu =     11.5    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     12.4    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.6760239            2.023151            F
         3        -44.6377381            2.617887            F
         4        -44.6242679            2.176422            T

     path length          =  5.191 bohr
     inter-image distance =  1.730 bohr

     ------------------------------ iteration   8 ------------------------------

     tcpu =     12.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     13.6    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.8337576            0.259579            F
         3        -44.7229591            5.380810            F
         4        -44.6242679            2.176422            T

     path length          =  5.446 bohr
     inter-image distance =  1.815 bohr

     ------------------------------ iteration   9 ------------------------------

     tcpu =     14.4    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     15.2    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7999787            1.086581            F
         3        -44.7213902            1.203266            F
         4        -44.6242679            2.176422            T

     path length          =  5.351 bohr
     inter-image distance =  1.784 bohr

     ------------------------------ iteration  10 ------------------------------

     tcpu =     15.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     16.5    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.8207205            0.506238            F
         3        -44.7367109            1.302529            F
         4        -44.6242679            2.176422            T

     path length          =  5.469 bohr
     inter-image distance =  1.823 bohr

     ------------------------------ iteration  11 ------------------------------

     tcpu =     17.2    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     17.9    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7930466            1.130620            F
         3        -44.7435984            0.966719            F
         4        -44.6242679            2.176422            T

     path length          =  5.567 bohr
     inter-image distance =  1.856 bohr

     ------------------------------ iteration  12 ------------------------------

     tcpu =     18.4    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     19.0    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.8141058            0.368347            F
         3        -44.7438499            0.680448            F
         4        -44.6242679            2.176422            T

     path length          =  5.509 bohr
     inter-image distance =  1.836 bohr

     ------------------------------ iteration  13 ------------------------------

     tcpu =     19.6    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     20.2    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.8007606            0.274391            F
         3        -44.7571980            1.208768            F
         4        -44.6242679            2.176422            T

     path length          =  5.475 bohr
     inter-image distance =  1.825 bohr

     ------------------------------ iteration  14 ------------------------------

     tcpu =     20.8    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     21.4    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.8036829            0.277959            F
         3        -44.7508183            0.166160            F
         4        -44.6242679            2.176422            T

     path length          =  5.458 bohr
     inter-image distance =  1.819 bohr

     ------------------------------ iteration  15 ------------------------------

     tcpu =     22.0    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     22.6    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.8000452            0.276602            F
         3        -44.7499284            0.071324            F
         4        -44.6242679            2.176422            T

     path length          =  5.449 bohr
     inter-image distance =  1.816 bohr

     ------------------------------ iteration  16 ------------------------------

     tcpu =     23.2    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     23.7    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7907679            0.251206            F
         3        -44.7508372            0.131716            F
         4        -44.6242679            2.176422            T

     path length          =  5.432 bohr
     inter-image distance =  1.811 bohr

     ------------------------------ iteration  17 ------------------------------

     tcpu =     24.3    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     25.0    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7669638            0.176950            F
         3        -44.7531880            0.478084            F
         4        -44.6242679            2.176422            T

     path length          =  5.395 bohr
     inter-image distance =  1.798 bohr

     broyden uphill step : history is reset


     ------------------------------ iteration  18 ------------------------------

     tcpu =     25.6    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     26.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7659964            0.181225            F
         3        -44.7558773            0.089730            F
         4        -44.6242679            2.176422            T

     path length          =  5.389 bohr
     inter-image distance =  1.796 bohr

     ------------------------------ iteration  19 ------------------------------

     tcpu =     26.7    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     27.4    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7641975            0.172903            F
         3        -44.7566529            0.045161            F
         4        -44.6242679            2.176422            T

     path length          =  5.384 bohr
     inter-image distance =  1.795 bohr

     ------------------------------ iteration  20 ------------------------------

     tcpu =     27.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     28.5    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7585708            0.158289            F
         3        -44.7582033            0.090979            F
         4        -44.6242679            2.176422            T

     path length          =  5.374 bohr
     inter-image distance =  1.791 bohr

     ---------------------------------------------------------------------------


     neb: reached the maximum number of steps

     NEB          :     26.90s CPU     29.13s WALL


   This run was terminated on:  21: 1:35   9Sep2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
